apps/src/templates/instructions/InstructionsWithWorkspace.jsx

```diff
@@ -22,6 +22,11 @@ var InstructionsWithWorkspace = React.createClass({
     instructionsMaxHeight: React.PropTypes.number.isRequired,
     setInstructionsHeight: React.PropTypes.func.isRequired,
     setInstructionsMaxHeight: React.PropTypes.func.isRequired,
+
+    isCsf: React.PropTypes.bool,
```

Comment:
`isCsf` doesn't strike me as particularly descriptive.  Can prop be renamed to better represent what it actually affects?